app/main/views/suppliers.py

```diff
@@ -183,6 +163,16 @@ def approve_agreement_for_countersignature(agreement_id):
     ))
 
 
+@main.route('/suppliers/<supplier_id>/agreement/<framework_slug>', methods=['GET'])
+@login_required
+@role_required('admin', 'admin-ccs-sourcing')
+def download_signed_agreement_file(supplier_id, framework_slug):
+    # This route is used for pre-G-Cloud-8 agreement document downloads
```

Comment:
Could be good for this comment to indicate the relationship of this route with the `download_agreement_file` route, i.e. why does this route exist and what is the difference between the two in terms of usage